ios/Sources/Utility/Extension/Date+.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension Date {
+    public var formatted: String {
+        let formatter = DateFormatter.shared
+        formatter.dateStyle = .medium
+        formatter.timeStyle = .none
+        formatter.locale = Locale(identifier: "ja_JP")
+        return formatter.string(from: self)
+    }
+}
+
+extension DateFormatter {
+    static let shared = DateFormatter()
+}
```

Comment:
How about place this file to Component module?
I think `DateFormat` is part of UI appearance because this file should be localized.